roles/openshift_storage_glusterfs/tasks/gluster_s3_deploy.yml

```diff
@@ -111,3 +156,15 @@
   - "gluster_s3_pod.results.results[0]['items'] | lib_utils_oo_collect(attribute='status.conditions') | lib_utils_oo_collect(attribute='status', filters={'type': 'Ready'}) | map('bool') | select | list | count == 1"
   delay: 10
   retries: "{{ (glusterfs_timeout | int / 10) | int }}"
+
+- name: Delete GlusterFS StorageClass for S3
+  oc_obj:
+    namespace: "{{ glusterfs_namespace }}"
+    kind: "{{ item.kind }}"
+    name: "{{ item.name | default(omit) }}"
+    selector: "{{ item.selector | default(omit) }}"
+    state: absent
+  with_items:
```

Comment:
No apparent usage for with_items here.

Reply:
Holdover from a pattern I copy/pasted. Fixing.